configure-local-networking.sh

```diff
@@ -31,19 +33,43 @@ if $(which dnf >/dev/null 2>&1); then
     sudo dnf -y install iptables
 fi
 
+if [[ $ID = "ubuntu" ]]; then
+    sudo apt update
+    sudo apt -y install ifupdown
+fi
+
 # Configure local networking.
 # Add a bridge 'braio' for the Kayobe all-in-one cloud network.
 if ! sudo ip l show braio >/dev/null 2>&1; then
   sudo ip l add braio type bridge
-  sudo ip l set braio up
-  sudo ip a add $seed_hv_ip/24 dev braio
+  if [[ $ID = "ubuntu" ]]; then
+    cat << EOF | sudo tee /etc/network/interfaces.d/ifcfg-braio
+auto braio
+iface braio inet static
+address $seed_hv_ip/24
```

Comment:
`/24` is not needed since the netmask is defined on the next line. This should match the syntax used by MichaelRigart.interfaces.

Reply:
I'll update, although hopefully this goes away when we switch to networkd.